RELEASE.md

```diff
@@ -1,3 +1,59 @@
+# Release 2.3.0
+
+The 2.3 minor series tracks TensorFlow 2.3.
+
+## Features
+
+- The 30 sec default reload period is now customizable in the Settings UI (#2794)
+- 🧪 **Experimental** Debugger V2 is now available; see the [tutorial][debugger-v2-tutorial] on how to use the experimental TensorFlow APIs to spot NaN's in tensors, view graphs generated from executions, and the related lines in the Python source code (#3821)
+
+## TensorBoard.dev updates
+- Added support for showing the Distributions tab (#3762)
+- Uploader now displays data statistics in the console while uploading data (#3678)
+- Added new uploader command line flags (#3707)
+  - `--dry_run`: causes the uploader to only read the logdir and display statistics (if `--verbose` is the default 1) without uploading any data to the server
+  - `--one_shot`: causes the uploader to exit immediately after all existing data in the logdir are uploaded; this mode prints a warning message if the logdir doesn't contain any uploadable
+data
+- Upload button in the header offers a convenient, copyable command
+- 🧪 **Experimental** DataFrame API: You can now read Scalars data from TensorBoard.dev as a Pandas DataFrame (learn more [here][dataframe-tutorial]).
+
+[debugger-v2-tutorial]: https://www.tensorflow.org/tensorboard/debugger_v2
+[dataframe-tutorial]: https://www.tensorflow.org/tensorboard/dataframe_api
+
+## Bug fixes
+- Projector plugin
+  - Shows data when logs exist in both logdir root and subdirectory (#3694)
+  - Fixed incorrect embeddings from TF2 checkpoints (#3679)
+  - Added support for binary format, with 2x speedup loading large tensors in some cases (#3685) - thanks [@RustingSword](https://github.com/RustingSword)
+  - Added [Colab tutorial][projector-colab] for Projector plugin (#3423)
+- Notebooks
+  - Increased port scanning from 10 to 100 to better support multi-tenant Notebooks (#3780) - thanks [@jerrylian-db](https://github.com/jerrylian-db)
+  - Update magics in tutorial notebooks (#3459) - thanks [@ManishAradwad](https://github.com/ManishAradwad)
```

Comment:
This is a web docs change, which is definitely valuable but isn’t really
being released here, since the docs have been on stable for a while, and
it doesn’t really have to do with notebook support like its siblings.
If you want to include it, maybe merge into “Updated broken docs” below?

Reply:
Sounds good, I've merged into the `Updated broken docs`